examples/CMakeLists.txt

```diff
@@ -38,9 +38,7 @@ add_subdirectory(lapd-drift)
 add_subdirectory(laplacexy/alfven-wave)
 add_subdirectory(laplacexy/laplace_perp)
 add_subdirectory(laplacexy/simple)
-if(BOUT_HAS_HYPRE)
-  add_subdirectory(laplacexy/simple-hypre)
-endif ()
+add_subdirectory(laplacexy/simple-hypre)
```

Comment:
Will this example build without Hypre?

Reply:
Ah I see it's replaced by a `REQUIRES BOUT_HAS_HYPRE` clause to `bout_add_example`

Reply:
Yes, exactly, so the example gets skipped if Hypre isn't present